Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyConfigurations/Functionality - Windows Known Folders Move to OneDrive.json

```diff
@@ -62,5 +62,5 @@
       ]
     }
   ],
-  "description": "Configures Windows Known Folders Move to OneDrive and disable opt out of that feature. Also enables silent move of known folders to OneDrive."
+  "description": "What is it: Configures Windows Known Folders Move to OneDrive and disables the option to opt out the feature. \n Learn More: https://docs.microsoft.com/en-us/onedrive/redirect-known-folders"
```

Comment:
**What does this do?**
Configures Windows Known Folders Move to OneDrive and disables the option to opt out the feature.

**Why should you use this?**
OneDrive backs up user data in and enables access from any device.

**What is the end user impact?**
Allows users to continue using familiar folders while being backed up. 

**Learn more**
[Redirect and move Windows known folders to OneDrive](https://docs.microsoft.com/en-us/onedrive/redirect-known-folders)